README.md

```diff
@@ -109,10 +109,11 @@ Once you have [installed Docker](https://docs.docker.com/engine/installation/)
 just run the following command to get an environment that will run on either the cpu or gpu.
 
    ```bash
-   docker run -it -p 8000:8000 --rm allennlp/allennlp:v0.8.2
+   mkdir -p $HOME/.allennlp/
+   docker run --rm -v $HOME/.allennlp:/root.allennlp allennlp/allennlp:v0.8.2
```

Comment:
I'm not a Docker expert, is this really supposed to be `/root.allennlp` and not `/root/.allennlp`?